src/sources/ffmpeg.rs

```diff
@@ -0,0 +1,43 @@
+use songbird::input::{
+    error::{Error, Result},
+    Codec, Container, Input, Metadata, Reader,
+};
+use std::process::{Child, Command, Stdio};
+
+pub async fn ffmpeg_yt(mut yt: Child, metadata: Metadata, pre_args: &[&str]) -> Result<Input> {
+    let ffmpeg_args = [
+        "-f",
+        "s16le",
+        "-ac",
+        "2",
+        "-ar",
+        "48000",
+        "-acodec",
+        "pcm_f32le",
+        "-",
+    ];
```

Comment:
Can you go over these args? Why did we add them? Or were they already in use, but not specified by us?

Reply:
They were, please check: https://github.com/serenity-rs/songbird/blob/73323e58ddf47dfa2bb0e334c37e939cfbd95a86/src/input/ytdl_src.rs#L57-L67